cdap-common/src/test/java/co/cask/cdap/io/SchemaTest.java

```diff
@@ -149,6 +149,69 @@ public void testSchemaHash() throws UnsupportedTypeException {
   }
 
   @Test
+  public void testAvroSchemaEnum() throws Exception {
+   String schemaStr = "{ \"type\" : \"enum\",\n" +
```

Comment:
You can add a avro dependency (test scope) to the cdap-common if it is not there and use the Avro library to generate the schema rather than hand coding it.